src/containers/VaccineInfo/VaccineInfo.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  DisplayText,
+  Layout,
+  MediaCard,
+  VideoThumbnail,
+} from "@shopify/polaris";
+import { useTranslation } from "react-i18next";
+import React from "react";
+
+export function VaccineInfo() {
+  const { t } = useTranslation();
+  return (
+    <Layout>
+      <Layout.Section>
+        <DisplayText size="extraLarge">Vaccine Information</DisplayText>
+        <DisplayText size="small">
+          We have collected these resources for you to get more information on
+          vaccines.
+        </DisplayText>
+      </Layout.Section>
+      <Layout.Section>
+        <MediaCard
+          portrait
+          title="COVID-19: How vaccines are developed"
+          primaryAction={{
+            content: "Learn more",
+            url: "https://www.youtube.com/watch?v=Y51ZgZCS8J0",
```

Comment:
Links should open in new window use external: true

Reply:
Opening a new window can pose accessibility challenges: https://www.w3.org/TR/WCAG20-TECHS/G201.html